apps/minifront/package.json

```diff
@@ -39,17 +39,19 @@
     "@radix-ui/react-navigation-menu": "^1.1.4",
     "@radix-ui/react-portal": "^1.0.4",
     "@remix-run/router": "^1.16.1",
-    "@tanstack/react-query": "4.36.1",
+    "@skip-go/widget": "^3.0.22",
+    "@tanstack/react-query": "5.62.7",
```

Comment:
The react-query dep bump was required for the skip widget to work. Unfortunately, the osmo-query hooks only work on v4. Had to do some refactoring as a consequence, but it got simpler as a result.